custom_dc/README.md

```diff
@@ -193,6 +204,9 @@ before the operation.
 
 ### Prepare Custom Data
 
+> Note that the treatment of variables in this section is much simplified for the purposes of this discussion.
```

Comment:
It seems we do not introduce the word "variables" or "statistical variables" at all in this doc?  In the context of the CSV below, we say that `stat_var_1` and `stat_var_2` column names correspond to variables?  And then we can add this note.

Reply:
I think a separate high-level doc might be a better option to introduce concepts. There are a few that could be useful: variables, entities, dcid mapping / resolution, etc.

Make a suggestion if you want a short blurb in this doc.

Reply:
I feel inlining a one line intro (to avoid needing to read a new doc) is helpful

For example:

```
The first column header is a property that identifies the observed entity,
supported properties are `dcid`, `name`, `geoId`. When `dcid` is used, the
entity should have been resolved (from a previous step).

The second column is a property that identifies the date.

The last two columns identify IDs of metrics or statistical variables, with the column containing the corresponding numeric values.

> Note that the treatment of statistical variables is much simplified for the purposes of this discussion. For a more detailed discussion on the DC data model, please refer to the document on [Representing statistics in Data Commons](https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md).

```

Reply:
This is good. Applied it with a couple changes.